### PR TITLE
Add skonto to metrics reviewers

### PR DESCRIPTION
**Pull request by skonto**
Interested in reviewing PRs in this area.

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -60,6 +60,11 @@ aliases:
   - mdemirhan
   - yanweiguo
 
+  metrics-reviewers:
```
Comment (markusthoemmes) [inline OWNERS_ALIASES:63]: You'll have to add this group to the respective OWNER files as well please.

Comment (skonto) [reply OWNERS_ALIASES:63]: sure.

**Comment by skonto** [comment]
@markusthoemmes done.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**